src/Accelerators/NNPA/Conversion/ZHighToZLow/ZHighToZLow.cpp

```diff
@@ -200,6 +200,7 @@ Value insertAllocOrEmitZeroConstant(ArrayRef<IndexExpr> dims,
     int64_t sizeInBytes =
         affine::getIntOrFloatMemRefSizeInBytes(resType).value();
     char *rawData = (char *)malloc(sizeInBytes);
+    assert(rawData && "failed to allocate memory for stickified data");
```

Comment:
good idea, can probably systematically add asserts wherever the compiler use `malloc`

Reply:
Thanks!  Added some additional asserts